optimum/intel/neural_compressor/quantization.py

```diff
@@ -320,7 +324,23 @@ def from_pretrained(
 
                     raise EnvironmentError(msg)
 
-            state_dict = torch.load(state_dict_path)
+            try:
+                state_dict = torch.jit.load(state_dict_path)
+            except:
+                state_dict = torch.load(state_dict_path)
+        if isinstance(state_dict, torch.jit._script.RecursiveScriptModule):
+            assert example_inputs, "Please provide the example_inputs for IncQuantizedModel loading."
+            q_model = torch.jit.freeze(stat_dict.eval())
+            # After freezing, run 1 time to warm up the profiling graph executor to insert prim::profile
+            # At the 2nd run, the llga pass will be triggered and the model is turned int
+            # an int8 model: prim::profile will be removed and will have LlgaFusionGroup in the graph
+            if isinstance(example_inputs, torch.Tensor):
```

Comment:
It seems a little bit counterintuitive to have to give an example inputs to load the model, can we make it optional ? (and feed a dummy inputs by default ?)

Reply:
We now feed dummy inputs instead of example inputs. Here two inferences are required in order to replace all int8 ops.